src/bitcoinrpc.cpp

```diff
@@ -219,89 +256,305 @@ Value stop(const Array& params, bool fHelp)
+    { "getaddednodeinfo",        &getaddednodeinfo,        true,   cat_network       },
+    { "getbestblockhash",        &getbestblockhash,        true,   cat_network       },
+    { "getblock",                &getblock,                true,   cat_network       },
+    { "getblockbynumber",        &getblockbynumber,        true,   cat_network       },
+    { "getblockcount",           &getblockcount,           true,   cat_network       },
+    { "getblockhash",            &getblockhash,            true,   cat_network       },
+    { "getcheckpoint",           &getcheckpoint,           true,   cat_network       },
+    { "getconnectioncount",      &getconnectioncount,      true,   cat_network       },
+    { "getdifficulty",           &getdifficulty,           true,   cat_network       },
+    { "getinfo",                 &getinfo,                 true,   cat_network       },
+    { "getnettotals",            &getnettotals,            true,   cat_network       },
+    { "getpeerinfo",             &getpeerinfo,             true,   cat_network       },
+    { "getrawmempool",           &getrawmempool,           true,   cat_network       },
+    { "listallpolls",            &listallpolls,            true,   cat_network       },
+    { "listallpolldetails",      &listallpolldetails,      true,   cat_network       },
+    { "listpolldetails",         &listpolldetails,         true,   cat_network       },
+    { "listpollresults",         &listpollresults,         true,   cat_network       },
+    { "listpolls",               &listpolls,               true,   cat_network       },
+    { "memorypool",              &memorypool,              true,   cat_network       },
+    { "networktime",             &networktime,             true,   cat_network       },
+    { "ping",                    &ping,                    true,   cat_network       },
+#ifdef WIN32
+    { "reindex",                 &reindex,                 true,   cat_network       },
+    { "restart",                 &restart,                 true,   cat_network       },
+    { "restorepoint",            &restorepoint,            true,   cat_network       },
+#endif
+    { "showblock",               &showblock,               true,   cat_network       },
+    { "stop",                    &stop,                    true,   cat_network       },
+    { "vote",                    &vote,                    false,  cat_network       },
+    { "votedetails",             &votedetails,             true,   cat_network       }
 };
 
+template<typename T>
+void ConvertTo(Value& value, bool fAllowNull=false)
+{
+    if (fAllowNull && value.type() == null_type)
+        return;
+    if (value.type() == str_type)
+    {
+        // reinterpret string as unquoted json value
+        Value value2;
+        string strJSON = value.get_str();
+        if (!read_string(strJSON, value2))
+            throw runtime_error(string("Error parsing JSON:")+strJSON);
```

Comment:
Probably needs a space:
"Error parsing JSON:" should be 
"Error parsing JSON: "